L.Map.Sync.js

```diff
@@ -41,6 +48,27 @@
             return this;
         },
 
+        // Helper function to compute the offset easily
+        // The arguments are relative positions respect refence and target maps
+        // of the point to sync.
+        // If you provide ratioRef=[0,1], ratioTgt=[1,0] will sync
+        // the bottom left corner of the reference map
+        // with the top right corner of the target map.
+        // The values can be less than 0 or greater than 1. It will sync
+        // points out of the map.
+        offsetHelper: function (ratioRef, ratioTgt) {
```

Comment:
While i was coding it I was thinking on where to place it. Yes, it is an "static" function (in Java or C++ notation), or a functions factory. It does not interact with the instance at all. That lets you more freedom to use your own function if you want something fancier.
However, it is going to be used only in this context, to create the function expected in offsetFn.

Maybe makes sense to move it to `L.Util.offsetHelper`... or with a different name?